osu.Game/Utils/ModUtils.cs

```diff
@@ -112,14 +112,34 @@ public static bool CheckAllowed(IEnumerable<Mod> combination, IEnumerable<Type>
         /// <param name="invalidMods">Invalid mods, if any were found. Can be null if all mods were valid.</param>
         /// <returns>Whether the input mods were all valid. If false, <paramref name="invalidMods"/> will contain all invalid entries.</returns>
         public static bool CheckValidForGameplay(IEnumerable<Mod> mods, [NotNullWhen(false)] out List<Mod>? invalidMods)
+            => checkValid(mods, m => m.Type != ModType.System && m.HasImplementation && !(m is MultiMod), out invalidMods);
+
+        /// <summary>
+        /// Check the provided combination of mods are valid for a multiplayer match session.
+        /// </summary>
+        /// <param name="mods">The mods to check.</param>
+        /// <param name="invalidMods">Invalid mods, if any were found. Can be null if all mods were valid.</param>
```

Comment:
nit: weird to use "can" in a conditional clause

```suggestion
        /// <param name="invalidMods">Invalid mods, if any were found. Will be null if all mods were valid.</param>
```

goes for the other method too